docs/networks/software-versions.mdx

```diff
@@ -12,6 +12,47 @@ Release candidates are software releases that are also released to the [Testnet]
 
 [testnet]: ./README.mdx
 
+## Protocol 22 (Testnet, October , 2024)
+
+### Software
+
+| Software | Version |
+| --- | --- |
+| XDR | `v22.0` |
+| Rust XDR | `v22.0.0-rc.1.1` |
+| Soroban Environment | `v22.0.0-rc.2` |
+| Stellar Core | `v22.0.0rc2` |
+| Soroban Rust SDK | `v22.0.0-rc.2.1` |
+| Stellar CLI | `v21.0.0` |
```

Comment:
How about leaving the downstream v21 versions as TBD for now? As the teams release their versions they can update this.

Reply:
Good point. Done